java-checks/src/test/java/org/sonar/java/checks/RulesSanityTest.java

```diff
@@ -120,7 +120,7 @@ void test() throws Exception {
       .collect(Collectors.toList());
 
     assertThat(errorLogs)
-      .hasSize(6)
+      .hasSize(8)
```

Comment:
In test for Java 8 `yield()` method is allowed to be outside of switch expression, while in the recent versions of Java it doesn't parse.